pages/issues.py

```diff
@@ -39,3 +42,33 @@ def click_status_button(self, name_button):
             if name_button == element.text:
                 element.click()
                 return
+
+    def send_date_datepicker_to(self, day, month, year):
+        """
+            Method for send data in datepicker 'to'.
+            Day, month, year - is int value.
+        """
+        element = self.driver.find_element(By.CSS_SELECTOR, self.DATAPICKER_TO_CSS)
+        element.send_keys(year)
+        element.send_keys(year)
+        element.send_keys(year)
+        element.send_keys(Keys.ARROW_LEFT)
+        element.send_keys(month)
+        element.send_keys(Keys.ARROW_LEFT)
+        element.send_keys(Keys.ARROW_LEFT)
+        element.send_keys(day)
```

Comment:
Actually this code can be extracted to the separate method `write_date_to_datepicker` where you will pass `(element, year, month, day)`
And in this method will be only 
```suggestion
        element = self.driver.find_element(By.CSS_SELECTOR, self.DATAPICKER_TO_CSS)
        write_date_to_datepicker(element, year, month, day)
```
Or you may even pass not an element, but just a selector in the `write_date_to_datepicker` method

By doing this you will avoid code duplication

Reply:
Or it would be really cool to create a separate element class for datepicker and use it as you are going to use your future ButtonElements class
I vote for this option :smile:

Reply:
ok, I will create class DatePicker)

Reply:
I fix it after push 'Button' and 'DatePicker' Classes

Reply:
done